src/extension.ts

```diff
@@ -1027,3 +1039,10 @@ function configureSettingsDefaults() {
     ConfigurationTarget.Workspace
   );
 }
+
+function toggleBooleanWorkspaceSetting(setting: string) {
+  const config = workspace.getConfiguration("metals");
+  const configProperty = config.inspect<boolean>(setting);
+  const currentValues = configProperty?.workspaceValue ?? false;
+  config.update(setting, !currentValues, ConfigurationTarget.Workspace);
```

Comment:
@gabro I set it to use workspace by default, I think it makes sense for toggle. What do you think?